wwwroot/init/simple.json

```
@@ -751,5 +751,9 @@
     }
   ],
   "baseMapId": "basemap-bing-aerial-with-labels",
-  "previewBaseMapId": "basemap-positron"
+  "previewBaseMapId": "basemap-positron",
+  "viewerMode": "3dSmooth",
+  "baseMaps": {
+    "defaultBaseMapId": "basemap-positron"
```

Positron is the default preview basemap, but it's probably a good idea to show how to set the preview basemap in simple.json
```suggestion
    "defaultBaseMapId": "basemap-positron",
    "previewBaseMapId": "basemap-positron"
```